jax/experimental/jax2tf/impl_no_xla.py

```diff
@@ -792,10 +792,99 @@ def _dynamic_update_slice(operand, update, *start_indices,
 
 tf_impl_no_xla[lax.dynamic_update_slice_p] = _dynamic_update_slice
 
-tf_impl_no_xla[lax.scatter_p] = _unimplemented("scatter")
-tf_impl_no_xla[lax.scatter_min_p] = _unimplemented("scatter min")
-tf_impl_no_xla[lax.scatter_max_p] = _unimplemented("scatter max")
-tf_impl_no_xla[lax.scatter_mul_p] = _unimplemented("scatter mul")
-tf_impl_no_xla[lax.scatter_add_p] = _unimplemented("scatter add")
+def get_move_to_leading_axis_bijector(axis):
```

Comment:
I am a bit confused by this name and the docstring. Given that it is only used once, perhaps it is easier to understand if you simply define the `shift` function and then define `fwd = shift` and `bwd = partial(shift, inverse=True)` below where you need it?